src/scripts/crank.ts

```diff
@@ -34,24 +36,25 @@ const {
   MAX_UNIQUE_ACCOUNTS,
   CONSUME_EVENTS_LIMIT,
   CLUSTER,
-  HIGH_FEE_MARKETS,      // markets to apply a priority fee for
   PRIORITY_QUEUE_LIMIT,  // queue length at which to apply the priority fee
-  DEFAULT_CU_PRICE,     // extra microlamports per cu for any market
   PRIORITY_CU_PRICE,     // extra microlamports per cu for high fee markets
   PRIORITY_CU_LIMIT,     // compute limit
   POLL_MARKETS, // optional for using Top markets
+  MAX_TX_INSTRUCTIONS,   // max instructions per transaction
+  MIN_FEE_BUMP,          // extra microlamports per cu for any transaction
```

Comment:
let's use the same name please like before e.g. "default" cu_price and "priority" cu_price, that makes it clear the 2 vars are related. Either call both FEE_BUMP or CU_PRICE. Preferably CU_PRICE since we also use "CU_LIMIT"

this is cosmetic but it makes the code clearer

Reply:
https://github.com/SpaceMonkeyForever/openbook-cranker/pull/4/commits/f90ed91210998dd0475665f66f83840ee6eb5d24